web/public/css/header.less

```diff
@@ -133,26 +133,26 @@
 }
 
 .cd-color-1, html .container .cd-color-1 {
-  background-color: #61C93F;
+  background-color: #61C93F !important;
```

Comment:
Because it's used for panels in the "setup your dojo" step of the SAD process and it's overwritten by bootstrap which has a more strict rule ( panel>heading)
Best solution would be to do a loop for colors inside a mixin, but since SAD process must be reworked graphically anyway, that's a quickfix